src/libraries/System.Private.CoreLib/src/System/ThrowHelper.cs

```diff
@@ -575,6 +577,26 @@ internal static void ThrowFormatIndexOutOfRange()
             throw new FormatException(SR.Format_IndexOutOfRange);
         }
 
+        internal static AmbiguousMatchException GetAmbiguousMatchException(MemberInfo memberInfo)
+        {
+            Type? declaringType = memberInfo.DeclaringType;
+            return (declaringType is not null)
+                ? new AmbiguousMatchException(SR.Format(SR.Arg_AmbiguousMatchException_Detailed, declaringType.ToString(), memberInfo.ToString()))
+                : new AmbiguousMatchException(SR.Format(SR.Arg_AmbiguousMatchException_Simple, memberInfo.ToString()));
+        }
+
+        internal static AmbiguousMatchException GetAmbiguousMatchException(Attribute attribute)
+        {
+            Type attributeType = attribute.GetType();
+            return new AmbiguousMatchException(SR.Format(SR.Arg_AmbiguousMatchException_Detailed, attributeType.ToString(), attribute.ToString()));
+        }
+
+        internal static AmbiguousMatchException GetAmbiguousMatchException(CustomAttributeData customAttributeData)
+        {
+            Type attributeType = customAttributeData.GetType();
+            return new AmbiguousMatchException(SR.Format(SR.Arg_AmbiguousMatchException_Detailed, attributeType.ToString(), customAttributeData.ToString()));
```

Comment:
`.ToString()` is redundant here and everywhere else

Reply:
Snipped.